common/geth/handle_error.go

```diff
@@ -25,7 +25,7 @@ const (
 func (f *FailoverController) handleHttpError(httpRespError rpc.HTTPError) (NextEndpoint, ImmediateAction) {
 	sc := httpRespError.StatusCode
 	// Default to rotation the current RPC, because it allows a higher chance to get the query completed.
-	f.Logger.Info("[HTTP Response Error]", "Status Code", sc, "Error", httpRespError)
+	f.Logger.Info("[HTTP Response Error]", "Curr Rpc Fault", f.numberRpcFault, "Status Code", sc, "Error", httpRespError)
```

Comment:
Let's not use spaces as attribute labels
`f.Logger.Info("[HTTP Response Error]", "numRPCFfault", f.numberRpcFault, "statusCode", sc, "err", httpRespError)`